front/migrations/db/migration_04.sql

```diff
@@ -0,0 +1,2 @@
+-- Migration created on May 17, 2024
+CREATE INDEX "conversations_workspace_id" ON "conversations" ("workspaceId")
```

Comment:
I called it `conversations_wId_idx` on production.
papertrail: https://dust4ai.slack.com/archives/C050SM8NSPK/p1715940435573269

Reply:
Good catch, updated.